kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KyuubiApplicationManager.scala

```diff
@@ -106,6 +108,22 @@ object KyuubiApplicationManager {
     conf.set(FlinkProcessBuilder.TAG_KEY, newTag)
   }
 
+  private def checkSparkPathURIs(
+      appConf: Map[String, String],
+      sessionManager: KyuubiSessionManager): Unit = {
```

Comment:
it seems we can pass kyuubi conf and move `sessionManager.checkSessionAccessPathURI` into `KyuubiApplicationManager` ?

Reply:
the `SESSION_LOCAL_DIR_ALLOW_LIST` is server side only conf, so it can not passed to sessionConf.

Reply:
let me try.